documentation/modules/rn-2.5.adoc

```diff
@@ -41,29 +41,36 @@ The old UI of MTV 2.3 cannot be enabled by setting `feature_ui: true` in Forklif
+In {project-short} {project-version}, you can migrate using Open Virtual Appliance (OVA) files that were created by VMware vSphere as source providers. link:https://issues.redhat.com/browse/MTV-336[(MTV-336)]
 
-.Migrating VMs between OCP clusters
+[NOTE]
+====
+Migration of OVA files that were not created by VMware vSphere but are compatible with vSphere might succeed. However, migration of such files is not supported by {project-short}. {project-short} supports only OVA files created by VMware vSphere.
+====
+
+include::snippet_ova_tech_preview.adoc[]
+
+.Migrating VMs between {ocp} clusters
 
 // i need to wait for this ticket to be merged to add a link
 // apologies, I meant link to output of the doc ticket rather than https://issues.redhat.com/browse/MTV-336[Import VMware-compatible OVAs]
 // But let me add this as a placeholder
-In {project-short} {project-version}, you can also use Red Hat {virt} provider as a source provider. You can migrate VMs from the cluster that MTV is deployed on to another cluster, or from a remote cluster to the cluster that {project-short} is deployed on. link:https://issues.redhat.com/browse/MTV-571[MTV-571]
+In {project-short} {project-version}, you can now use Red Hat {virt} provider as a source provider as well as a destination provider. You can migrate VMs from the cluster that MTV is deployed on to another cluster, or from a remote cluster to the cluster that {project-short} is deployed on. link:https://issues.redhat.com/browse/MTV-571[(MTV-571)]
 
 .Migration of VMs with direct LUNs from RHV
 
-During the migration from RHV, direct LUNs are detached from the source virtual machines and attached to the target virtual machines. Note that this mechanism does not work yet for Fibre Channel. link:https://issues.redhat.com/browse/MTV-329[MTV-329]
+During the migration from RHV, direct LUNs are detached from the source virtual machines and attached to the target virtual machines. Note that this mechanism does not work yet for Fibre Channel. link:https://issues.redhat.com/browse/MTV-329[(MTV-329)]
 
 .Additional authentication methods for OpenStack
 
-In addition to the standard password authentication, the following authentication methods are supported: Token authentication and Application credential authentication. Workaround: The `regionName` property needs to be added manually to the provider secret when these new authentication methods are configured from the UI. link:https://issues.redhat.com/browse/MTV-539[MTV-539]
+In addition to standard password authentication, the following authentication methods are supported: Token authentication and Application credential authentication. Usage note: The `regionName` property needs to be added manually to the provider secret when these new authentication methods are configured from the UI. link:https://issues.redhat.com/browse/MTV-539[(MTV-539)]
```

Comment:
Please can I about a **Usage note** and a **Workaround**... for my own knowledge. 
I am not asking for a change

Reply:
@anarnold97 To my mind, "Workaround" suggests there's a problem in the product but there is a way to deal with it. "Usage note," is meant to sidestep the issue of defect -- here we have a CLI feature that currently doesn't have a UI parallel.  Is that a defect? Not sure.

This phrase is part creative, part a poor recall by me of "product recommendation," which we started using with OADP release notes.  If the point I'm making is too subtle, I'm fine with "workaround" -- WDYT?

Reply:
Cool -makes sense